### PR TITLE
Replace additional magic numbers #332

### PR DESCRIPTION
**Pull request by tortmayr**
<i><b>Note </b>that this pull request is based on pull request #321 for issued #46 #47 #56  and #57, which is not yet merged.</i>

Migrates additional magic numbers used for the Lifeline Header to the layout constraints.
This PR addresses #332

### CODE REVIEW

File: plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/internal/model/spi/impl/DefaultLayoutHelper.java [hunk 1]
```diff
@@ -801,9 +801,9 @@ public Bounds getNewBounds(EClass eClass, Bounds proposedBounds, Node container)
 				case UMLPackage.LIFELINE:
 					// TODO the magic number 25 is referenced from
```
Comment (planger) [inline plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/internal/model/spi/impl/DefaultLayoutHelper.java]: Please also remove those todos which now got irrelevant. Thanks!

**Comment by planger** [comment]
@tortmayr Unfortunately, we have to rebase your change after the other changes from @jfaltermeier have been merged now. Thanks!

**Comment by tortmayr** [comment]
@planger  I rebased this PR on the current master. In addition, I found some left-over magic numbers in DefaultDiagramHelper and migrated them as well.

**Comment by planger** [comment]
Great, thanks a lot for the fast rebase and additional fix!

**Pull request merged**